functions/src/sendgrid-key.json

```diff
@@ -0,0 +1 @@
+{"SENDGRID_API_KEY":""}
```

Comment:
this should not be here

Reply:
Is my CI code incorrect? I removed the file and the CI command to write a dummy file is still there but the CI is failing

Reply:
@jason-tung you need to add the CI command to all the jobs that might need the key. It's left as an exercise for the reader to figure out what are those CI jobs 😉

Reply:
🤔